src/components/new-request/search-components/business-lookup-fetch.vue

```diff
@@ -141,17 +142,19 @@ export default class BusinessLookupFetch extends Mixins(CommonMixin, SearchMixin
       if (this.getSearchBusiness) {
         if (this.isBcBenCccUlc) {
           this.setLocation(Location.BC)
-          const corpType = this.getSearchBusiness?.legalType as unknown as CorpTypeCd
+          const corpType = this.getSearchBusiness.legalType as unknown as CorpTypeCd
```

Comment:
Same comment as on line 130.

Reply:
Or maybe not, since you didn't use conditional chaining here?

Reply:
I don't think so in this case since getSearchBusiness will not be null here, and legalType will default to null if unassigned. A fallback will be needed for your comment on line 130 however.